development/introduction.md

```diff
@@ -30,6 +30,11 @@ We recommend the following pattern: `MMM-MyNewCoolModule`. But it is not
 mandatory - your module will also work with other name patterns (as long as no
 other module has the same name).
 
+## Quick start
+
+To rocket launch your module, you can use the [MMM-Template](https://github.com/Dennis-Rosenbaum/MMM-Template) or immediately  [MMM-Template](https://github.com/new?template_name=MMM-Template&template_owner=Dennis-Rosenbaum). 
+Please read the [MMM-Template wiki](https://github.com/Dennis-Rosenbaum/MMM-Template/wiki) for more information.
+
```

Comment:
could you add a note that this is an externally developed module and not maintained by the MagicMirrorOrg?

Reply:
@rejas I have added the note so it should now be clear for the users.